libs/shared/src/entities/subscriber/subscriber.interface.ts

```diff
@@ -0,0 +1,29 @@
+import { ChatProviderIdEnum, PushProviderIdEnum } from '../../consts';
+
+export interface ISubscriber {
+  _id?: string;
+  firstName: string;
+  lastName: string;
+  email: string;
+  phone?: string;
+  avatar?: string;
+  subscriberId: string;
+  channels?: IChannelSettings[];
+  _organizationId: string;
+  _environmentId: string;
+  deleted: boolean;
+  createdAt: string;
+  updatedAt: string;
+  __v?: number;
+}
+
+export interface IChannelSettings {
+  _integrationId: string;
+  providerId: ChatProviderIdEnum | PushProviderIdEnum;
+  credentials: IChannelCredentials;
+}
+
+export interface IChannelCredentials {
```

Comment:
It is great that you created this file!
Would you mind importing `IChannelCredentials` from this file to `packages/node/src/lib/subscribers/subscriber.interface.ts` ? As it uses the same type🚀

Reply:
Done ✅